backend/logistica_servicio/gestor_entregas/infraestructura/logistica_entregas.py

```
@@ -0,0 +1,122 @@
+        2. calcular la ruta agregando el destino nuevo, si no hay entregas existentes seria la ruta origen-destino
+        3. agregar la EntregasProgramadas con la nueva ruta calculada
+        4. agregar la EntregasProgramadasDetalle con la entregaProgramadaId y la entregaId
+        """
+        entregas_programadas = self.repositorio_entregas_programadas.obtener_entregas_programadas_por_fecha_camion(fecha, camion_id)
+        origen_coordenadas = self.convert_str_to_list(origen)
+        destino_coordenadas = self.convert_str_to_list(destino_coordenadas)
+        if not entregas_programadas:
+            request_data = {"punto_inicio": {"origen": origen_coordenadas}, "destinos": {destino_direccion: {"destino": destino_coordenadas}}}
+        else:
+            request_data = {"punto_inicio": {"origen": origen_coordenadas}, "destinos": {}}
+            for entrega in entregas_programadas:
```

Ruff (0.8.2)

Loop control variable `entrega` not used within loop body

Rename unused `entrega` to `_entrega`

(B007)